packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -82,6 +90,50 @@ class ExpansionPanel {
   /// The widget builder that builds the expansion panels' header.
   final ExpansionPanelHeaderBuilder headerBuilder;
 
+  /// The widget builder that builds a custom expansion icon to expand
+  /// the panel.
+  ///
+  /// If the widget builder is used, some form of custom gesture detection is
```

Comment:
This could be more explicit. For example: 
If [expandIconBuilder] is specified then the widget it returns must rebuild the exansion panel with a new [isExpanded] value.

Reply:
Done